composite_test.go

```diff
@@ -478,11 +479,11 @@ func TestUnionFileReaddirAskForTooMany(t *testing.T) {
 	overlay := &MemMapFs{}
 
 	for i := 0; i < 5; i++ {
-		WriteFile(base, fmt.Sprintf("file%d.txt", i), []byte("afero"), 0777)
+		WriteFile(base, fmt.Sprintf("/file%d.txt", i), []byte("afero"), 0777)
 	}
 
 	ufs := &CopyOnWriteFs{base: base, layer: overlay}
-
+	
```

Comment:
Can you please gofmt your code?

Reply:
right, must have misconfigured my file watcher, it's removed.